index.html

```diff
@@ -287,6 +293,7 @@ <h2 class="first-text">About <br /><span>Myself</span></h2>
                   <h2 class="javaS">JavaScript</h2>
                 </div>
               </li>
+              
```

Comment:
- [ ] You are almost there! Let's increase the space between this two columns, as shown in the template:

|Template|Your design|
|---|---|
|![Captura de pantalla 2022-06-01 a la(s) 9 15 04 p  m](https://user-images.githubusercontent.com/83717285/171537388-36083099-753b-49c2-a53e-890951d59244.png)|![Captura de pantalla 2022-06-01 a la(s) 9 27 14 p  m](https://user-images.githubusercontent.com/83717285/171537749-258cfc20-3bbf-40dd-b858-5e419d763097.png)|